### PR TITLE
PCHR-2464: Post amnesty merge fixes

### PR DESCRIPTION
**Pull request by AkA84**
## Overview
This PR contains a couple of fixes that were necessary after merging the `amnesty-styling` branch in Shoreditch (see #24 )

### Page content top padding
There was a problem involved with the following element in Drupal's `#branding` element:
```html
<div class="add-or-remove-shortcuts add-shortcut">
  <a href="/drupal7/admin/config/user-interface/shortcut/shortcut-set-1/add-link-inline link=...">
    <span class="icon"></span>
    <span class="text">Add to <em class="placeholder">Default</em> shortcuts</span>
  </a>
</div>
```
The element would be added to the DOM if the "Shortcuts" module was enabled, pushing the whole page down, and the page's `padding-top` was calculated based on this element to be always present

shortcuts-link-before

The problem occured when a site *didn't* have the "Shortcuts" module enabled. With that element missing, the whole page was pushed up, resulting in almost no top padding at all

no-shortcut-module-top-before

Bu setting `display: none` instead of (`visibility: hidden`) on the element, the fact that the element is there or not is inconsequential, allowing the top padding to be consistent

shortcuts-link-after

no-shortcut-module-top-after

### "New individual" page broken in CiviHR

new-individual-before

The problem here was simply that in [applying the patch](https://github.com/compucorp/org.civicrm.shoreditch/pull/24/commits/751749cd045da5a83ac126fb64ca76130e8a46e1) a selector was placed under the wrong parent selector. Placing the selector under the correct parent fixed the issue

new-individual-after

### Select2 ran on crm-form-multiselect widgets
The select2 plugin was enabled also on multiselect widgets, breaking them
```js
CRM.$('select:visible:not([class*="no-select2"])')
  .select2({
    containerCss: {
      display: 'inline-block'
    }
  })
```
multiselect-before

Simply filtering those out solved the issue
```js
CRM.$('select:visible:not([class*="no-select2"]):not([class*="crm-form-multiselect"])')
  .select2({
    containerCss: {
      display: 'inline-block'
    }
  })
```
multiselect-after

### CODE REVIEW

**Comment by igorpavlov-zz** [comment]
Please separate the whole description into 2 sections:
Overview (your first paragraph)
Technical Description (the rest)

P.S. (clicked wrong button, didn't mean to close PR)

**Comment by igorpavlov-zz** [comment]
> The element would be added to the DOM if the "Shortcuts" module was enabled

> The select2 plugin was enabled also on multiselect widgets, breaking them

Any chance we can test if other modules/widgets are enabled/disabled? Might it be a good idea?

**Comment by AkA84** [comment]
@igorpavlov from what we've seen with the backstopjs tests no other modules influence the appearance of the page

File: js/enable-select2.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ CRM.$(function() {
    * during DOM changes
    */
   var observer = new MutationObserver(debounce(function() {
-    CRM.$('select:visible:not([class*="no-select2"])')
+    CRM.$('select:visible:not([class*="no-select2"]):not([class*="crm-form-multiselect"])')
```
Comment (igorpavlov-zz) [inline js/enable-select2.js]: Why not :not(.crm-form-multiselect), is it a special selector syntax?

**Review by igorpavlov-zz — CHANGES_REQUESTED** [review]

**Review by igorpavlov-zz — APPROVED** [review]

**Pull request merged**